src/scheduler/run.py

```
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import subprocess
+from argparse import ArgumentParser
+for deps_path in [join(sep, "usr", "share", "bunkerweb", *paths) for paths in (("deps", "python"), ("utils",), ("api",), ("db",))]:
```

Ruff (0.8.2)

Undefined name `join`

(F821)